### ISSUE TITLE
About friend dialog is misleading

### ISSUE THREAD

**Issue by sphaerophoria** [qTox/qTox#5379]
Currently on the AboutFriend dialog we have OK and Cancel buttons at the bottom. This is misleading as it indicates to the user that if they press cancel the settings they change will not be applied. This is not the case as we apply settings as soon as they have been touched.

**Comment by jakieoymj-github** [comment]
agree

**Comment by sudden6** [comment]
Maybe renaming `Cancel` to `Close` would be a solution to this?

**Comment by sphaerophoria** [comment]
> Maybe renaming `Cancel` to `Close` would be a solution to this?

I'm okay with that if we also remove the OK button. A single close button indicates intent better IMO.

**Comment by Diadlo** [comment]
@sudden6 Not really. As @sphaerophoria said, there is 2 buttons which do the same thing. I think, name doesn't matter here

**Comment by sphaerophoria** [comment]
> 
> 
> @sudden6 Not really. As @sphaerophoria said, there is 2 buttons which do the same thing. I think, name doesn't matter here

Welllll... They do behave slightly differently in a couple cases which I can't remember off the top of my head, but for the vast majority they do the same thing.

**Comment by jakieoymj-github** [comment]
How about one Apply, one Close? Apply button default disabled, alternate button enabled or disabled while editting

**Comment by sphaerophoria** [comment]
> How about one Apply, one Close? Apply button default disabled, alternate button enabled or disabled while editting

I vote on just making everything apply on set as a fixup, apply vs cancel/close will require significantly more work.

**Comment by sudden6** [comment]
I agree with @sphaerophoria instant apply seems better for me too. @Diadlo if they are practically the same thing it would make sense to remove one of them.
